packages/local_auth/local_auth/test/local_auth_test.dart

```diff
@@ -40,28 +40,14 @@ void main() {
         expect(
           log,
           <Matcher>[
-            isMethodCall(
-              'authenticate',
-              arguments: <String, dynamic>{
-                'localizedReason': 'Needs secure',
-                'useErrorDialogs': true,
-                'stickyAuth': false,
-                'sensitiveTransaction': true,
-                'biometricOnly': true,
-                'biometricHint': androidBiometricHint,
-                'biometricNotRecognized': androidBiometricNotRecognized,
-                'biometricSuccess': androidBiometricSuccess,
-                'biometricRequired': androidBiometricRequiredTitle,
-                'cancelButton': androidCancelButton,
-                'deviceCredentialsRequired':
-                    androidDeviceCredentialsRequiredTitle,
-                'deviceCredentialsSetupDescription':
-                    androidDeviceCredentialsSetupDescription,
-                'goToSetting': goToSettings,
-                'goToSettingDescription': androidGoToSettingsDescription,
-                'signInTitle': androidSignInTitle,
-              },
-            ),
+            isMethodCall('authenticate',
```

Comment:
Tests in the app-facing package need to be rewritten to mock the platform interface, not the method channel. Whether any given implementation uses this platform channel format is an implementation detail of what is now a separate package.

Reply:
Yes, this has been done in #4701 which contains all changes for the app facing package.